elliottlib/openshiftclient.py

```diff
@@ -0,0 +1,24 @@
+"""
+Utility functions and object abstractions for general interactions
+with oc cli
+"""
+
+# stdlib
+from subprocess import call, check_output, CalledProcessError
+
+# ours
+from elliottlib.exceptions import ElliottFatalError
+
+
+def get_bug_list(working_dir, old, new):
+    bug_list = []
+    try:
+        bug_list = check_output(['oc', 'adm', 'release', 'info',
+                                '-o', 'name', # only output BZ IDs
+                                '--bugs={}/origin'.format(working_dir), # clone origin to working dir
```

Comment:
Does working dir get wiped away every time? We may want to think about adding that as a parameter in `~/.config/elliott/settings.yaml` so we can have it in a persistent spot? You will know better though since you're closer to this than I am right now.

Reply:
I think it already is something you can put in there. I will double check.